Uplift/Utils/Constants.swift

```diff
@@ -126,6 +126,7 @@ struct Constants {
         static let gymDetailHorizontal: CGFloat = 24
         static let gymDetailSpacing: CGFloat = 20
         static let homeHorizontal: CGFloat = 16
+        static let tabBarHeight: CGFloat = 64
     }
 
     /// Shadows usde in Uplift's design system.
```

Comment:
Wait can you fix this spelling error lmao. I just noticed this "usde"

Reply:
Lol yes I can change that